Surface_mesh_simplification/include/CGAL/Surface_mesh_simplification/Policies/Edge_collapse/Bounded_normal_change_placement.h

```diff
@@ -0,0 +1,94 @@
+// Copyright (c) 2017  GeometryFactory (France). All rights reserved.
+//
+// This file is part of CGAL (www.cgal.org).
+// You can redistribute it and/or modify it under the terms of the GNU
+// General Public License as published by the Free Software Foundation,
+// either version 3 of the License, or (at your option) any later version.
+//
+// Licensees holding a valid commercial license may use this file in
+// accordance with the commercial license agreement provided with the software.
+//
+// This file is provided AS IS with NO WARRANTY OF ANY KIND, INCLUDING THE
+// WARRANTY OF DESIGN, MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE.
+//
+// $URL$
+// $Id$
+//
+// Author(s)     : Andreas Fabri
+//
+#ifndef CGAL_SURFACE_MESH_SIMPLIFICATION_POLICIES_EDGE_COLLAPSE_BOUNDED_NORMAL_CHANGE_PLACEMENT_H
+#define CGAL_SURFACE_MESH_SIMPLIFICATION_POLICIES_EDGE_COLLAPSE_BOUNDED_NORMAL_CHANGE_PLACEMENT_H
+
+#include <CGAL/license/Surface_mesh_simplification.h>
+
+
+namespace CGAL {
+
+namespace Surface_mesh_simplification  
+{
+
+  template<class Placement>
+  class Bounded_normal_change_placement
+{
+public:
+    
+  typedef typename Placement::ECM ECM ;
+  
+public:
+  
+  Bounded_normal_change_placement(const Placement& placement = Placement() )
+    : mPlacement(placement)
+  {}
+     
+  template <typename Profile> 
+  optional<typename Profile::Point>
+  operator()( Profile const& aProfile) const
+  {
+    optional<typename Profile::Point> op = mPlacement(aProfile); 
```

Comment:
I added 
`/* triangles returns the triangles of the star of the vertices of the edge to collapse. 
 First the two trianges incident to the edge, then the other triangles. 
 The second vertex of each triangle is the vertex that gets placed */`